### PR TITLE
Change podnapisi URL to .eu

### PR DESCRIPTION
**Pull request by fernandog**
PR in behalf of @medariox

 The issue is that .net redirects to https and the default setting for allow_redirects (requests) is False for podnapisi. So we have to avoid the redirect by using https with .net or http with .eu, or set allow_redirects=True in podnapisi provider.

### CODE REVIEW

**Comment by fernandog** [comment]
@Diaoul why travis failed? Can't understand why

**Comment by fernandog** [comment]
@Diaoul can you comment please?

**Comment by Diaoul** [comment]
Let's use HTTPS instead.

**Comment by fernandog** [comment]
@Diaoul done

**Comment by fernandog** [comment]
@Diaoul can you check this please?

File: subliminal/providers/podnapisi.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ class PodnapisiProvider(Provider):
     """Podnapisi Provider."""
     languages = ({Language('por', 'BR'), Language('srp', script='Latn')} |
                  {Language.fromalpha2(l) for l in language_converters['alpha2'].codes})
-    server_url = 'http://podnapisi.net/subtitles/'
+    server_url = 'http://podnapisi.eu/subtitles/'
```
Comment (Diaoul) [inline subliminal/providers/podnapisi.py]: Shoudln't that be `https://podnapisi.net/subtitles/`?

**Pull request merged**